CMakeLists.txt

```diff
@@ -323,5 +347,5 @@ rocm_create_package(
     MAINTAINER "AMDMIGraphX Maintainer <migraphx-lib.support@amd.com>"
     LDCONFIG
     PTH
-    DEPENDS miopen-hip rocblas hip-rocclr hip-base half
+    DEPENDS miopen-hip rocblas hip-rocclr hip-base half libtbb2
```

Comment:
This actually needs to be added conditionally. Might skip it for this PR, but open an issue to track this. The issue is that we detect TBB in the `src` directory so we need to propagate this up to this directory. There is probably a way, but it might be more work to figure out so it would probably be best is a seperate PR.

Reply:
Understood. Created  #2565